code/modules/chemistry/Reagents-Misc.dm

```diff
@@ -4201,26 +4201,31 @@ datum
 			taste = "like a good quality all wear garment"
 			reagent_state = LIQUID
 
+			var/datum/material/jeanMaterial = null
+
+			New()
+				. = ..()
+				jeanMaterial = getMaterial("jean")
```

Comment:
I wouldn't do this, just because it might cause issues with GC or init order. Better to use `getMaterial` in the proc, since that call is essentially free anyway.